packages/govuk-frontend/src/govuk/components/accordion/accordion.puppeteer.test.js

```diff
@@ -1,12 +1,17 @@
 const {
-  goToComponent,
   goToExample,
   render,
   getAccessibleName
 } = require('@govuk-frontend/helpers/puppeteer')
 const { getExamples } = require('@govuk-frontend/lib/components')
 
 describe('/components/accordion', () => {
+  let examples
+
+  beforeAll(async () => {
+    examples = await getExamples('accordion')
+  })
```

Comment:
Unlike using `goToComponent()` via the Review app, we need all tests to have `examples` available

You'll see some test files had the `getExamples()` call in multiple places, but now they're at the top